### PR TITLE
Add score history

### PR DESCRIPTION
**Pull request by shynonagons**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -67,6 +67,9 @@ const getNumOfSameDiceValues = sortedArr => {
 // TO DO
 const getSameArrElements = (arrToSearch, arrSearchItems) => {};
 
+// helper function to map score values to labels
+const mapScoreLabel = (value) => (value[0].toUpperCase() + value.slice(1)).replace(/-/g, ' ');
+
```
Comment (vkWeb) [inline js/index.js:72]: Can't we use `Game.scoreHistory` for this?

File: js/index.js [hunk 2]
```diff
@@ -119,6 +122,14 @@ class Game {
     document.querySelector(
       "#current-round-rolls"
     ).textContent = this.rollsInCurrentRound;
+    const scoreboard = document.querySelector("#score-history");
+    scoreboard.textContent = '';
```
Comment (vkWeb) [inline js/index.js:126]: @perezvon Instead of resetting, what about printing the latest key / value of `Game.scoreHistory`?

**Review by vkWeb — COMMENTED** [review]
@perezvon I'm sorry for replying late to this. Things have been crazy lately. Thanks a lot for the work on this :) 

It would be amazing if you can address some of my concerns in another PR. The comments are mainly to optimize the approach.

File: js/index.js (hunk 1 above)
Comment (shynonagons) [reply js/index.js:72]: Well, we are; the `mapScoreLabel` function just converts the object key into a more legible string.

File: js/index.js (hunk 2 above)
Comment (shynonagons) [reply js/index.js:126]: See #9

File: js/index.js (hunk 1 above)
Comment (vkWeb) [reply js/index.js:72]: Yes, you are right.

File: js/index.js (hunk 2 above)
Comment (vkWeb) [reply js/index.js:126]: I will look into that.

**Pull request merged**